generator_test.go

```diff
@@ -264,6 +264,38 @@ func TestGenerateResponseData(t *testing.T) {
 			data.(map[string]interface{})["customer"])
 	}
 
+	// data replacement on `POST`
+	{
+		generator := DataGenerator{testSpec.Components.Schemas, &testFixtures}
+		data, err := generator.Generate(&GenerateParams{
+			RequestData: map[string]interface{}{
+				"customer": "ch_9999",
```

Comment:
really minor but that looks like a charge id

Reply:
Oops. Yep, thanks for that! My fingers just typed the wrong combination of letters. Fixed.